test/powershell/engine/Basic/DefaultCommands.Tests.ps1

```diff
@@ -516,7 +516,6 @@ Describe "Verify approved aliases list" -Tags "CI" {
 "Cmdlet",       "Write-Progress",                   "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "None"
 "Cmdlet",       "Write-Verbose",                    "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "None"
 "Cmdlet",       "Write-Warning",                    "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "None"
-"Cmdlet",       "Get-Subsystem",                    "",                                 $(             $CoreWindows -or $CoreUnix),     "",                     "",                     "None"
```

Comment:
Is the test fail?
If we plan to split SMA on subsystems I believe we need to improve the test to support subsystems. Maybe explicitly load subsystems and related cmdlets?

Reply:
The test failed at one time, and that was why I made this change. But it fails again with the recent commits and it turns out it doesn't care the cmdlets guarded by experimental features, so I reverted the changes to it.
This test will surely be dealt with when we start working on separating the first component.